portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,29 @@
     <script src="script.js"></script>
```

Comment:
You may want to change the <title> tag to something like "Nicolas's Portfolio" to make the tab title more descriptive